packages/js/src/types.ts

```diff
@@ -77,8 +76,16 @@ export interface Provider {
   models: ModelInfo[]
 }
 
+export interface CalcPrice {
+  input_price: number
+  output_price: number
+  total_price: number
+}
+
 export interface PriceCalculation {
-  price: number
+  input_price: number
+  output_price: number
+  total_price: number
```

Comment:
I'm fine to use camel case here, but we obviously can't change the interface to the providers data.

Reply:
fyi @petyosi

Reply:
That makes sense.

Reply:
upd: agreed not to make this update to have a good consistency with python and data.json.